src/db/commit.rs

```diff
@@ -183,7 +183,11 @@ impl Commit {
         let commit = commit_fb::Commit::create(&mut builder, commit_args);
         builder.finish(commit, None);
 
-        let chunk = dag::Chunk::new(builder.collapse(), &[value_hash]);
+        let refs = std::iter::once(value_hash)
+            .chain(basis_hash.into_iter())
```

Comment:
we need to decide whether to include localmeta.original_hash in the refs

Reply:
I'm trying to page all of this back into memory... Do we need to be able to read data from the original hash?